web-animations-1/Overview.bs

```diff
@@ -4809,10 +4809,10 @@ interface KeyframeEffect : AnimationEffect {
 :   <dfn attribute for=KeyframeEffect>pseudoElement</dfn>
 ::  The [=target pseudo-selector=].
     This is `null` if the [=effect target=] is an {{Element}} or is itself `null`.
-    When not `null`, this must be a valid [=pseudo-element=] selector,
-    attempting to set it to an invalid value will have no effect.
-    and will always return using the the double-colon syntax
-    (i.e. always `::before` and never `:before`).
+    When not `null`, this must be a [=pseudo-element=] selector with valid 
+    [[selectors-4#pseudo-element-syntax]].
+    This will not accept invalid values and will always return the selector in
+    its canonical form (i.e. `::before` instead of `:before`).
```

Comment:
I think this isn't strict enough. The linked-to syntax section still allows the single colon format whereas I think we want to explicitly reject that for this API since we don't want to propagate that form any further.

Reply:
I was aiming for consistency with other specs which accept pseudo-element selectors. Most I've encountered so far will accept but never return the legacy syntax. Should the question of whether to change that convention be handled here or should that be a separate issue/PR?

Reply:
@stephenmcgruer

Reply:
Yes, please file a separate issue for this. I seem to remember getting advice at one point that we shouldn't accept the old syntax either.